source/_integrations/cambridge_audio.markdown

```diff
@@ -0,0 +1,64 @@
+---
+title: Cambridge Audio
+description: Instructions on how to integrate Cambridge Audio Receivers into Home Assistant.
+ha_category:
+  - Media player
+ha_release: '2024.10'
+ha_iot_class: Local Push
+ha_domain: cambridge_audio
+ha_platforms:
+  - media_player
+ha_codeowners:
+  - '@noahhusby'
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The **Cambridge Audio** {% term integration %} allows you to control all receivers and streamers that support the StreamMagic app.
+
+The integration automatically discovers all enabled zones and sources. Each zone is added as a media player device with the enabled sources available as inputs. Media information and transport controls are supported if the selected source reports it.
```

Comment:
What are transport controls?

Reply:
Controls like play, pause, skip, previous, and seek